### PR TITLE
idfv_uuid_link updates

### PR DESCRIPTION
**Pull request by bkasman95**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/disabling_tracking.md [hunk 1]
```diff
@@ -14,3 +14,6 @@ To comply with data privacy regulations, data tracking activity on the iOS SDK c
 Additionally, you can use the method [`wipeDataAndDisableForAppRun`](http://appboy.github.io/appboy-ios-sdk/docs/interface_appboy.html#ac8d580f60ec0608cd91240a8a3aa23a3) to fully clear all client-side data stored on the device.
 
 Unless a user uninstalls all apps from a vendor on a given device, the next Braze SDK and app runs after calling `wipeDataAndDisableForAppRun()` will result in our server re-identifying that user via their device identifier (IDFV). In order to fully remove all user data, you should combine a call to `wipeDataAndDisableForAppRun` with a request to delete data on the server via the Braze [REST API]({{site.baseurl}}/developer_guide/rest_api/user_data/#user-delete-endpoint).
+
+## Swift v5.7.0+
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/disabling_tracking.md:18]:
```suggestion
## iOS SDK v5.7.0+
```

Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/disabling_tracking.md:18]:
```suggestion
## iOS SDK v5.7.0+
```

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/disabling_tracking.md [hunk 2]
```diff
@@ -14,3 +14,6 @@ To comply with data privacy regulations, data tracking activity on the iOS SDK c
...
+For devices using Swift SDK v5.7.0 and above, when [disabling IDFV collection](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/), calling [`wipeData`](https://braze-inc.github.io/braze-swift-sdk/documentation/brazekit/braze/wipedata()) will not result in our server re-identifying that user via their device identifier (IDFV).
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/disabling_tracking.md:19]:
```suggestion
For devices using iOS SDK v5.7.0 and above, when [disabling IDFV collection](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/), calling [`wipeData`](https://braze-inc.github.io/braze-swift-sdk/documentation/brazekit/braze/wipedata()) will not result in our server re-identifying that user via their device identifier (IDFV).
```

File: _docs/_developer_guide/platform_integration_guides/sdk_primer.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ These properties are collected by the iOS SDK upon proper integration.
 | Device Locale <br> `ABKDeviceOptionLocale`| The default locale of the device. |
 | Device Model <br> `ABKDeviceOptionModel`| The specific hardware of the device.
 | Device OS Version <br> `ABKDeviceOptionOSVersion` | The version of the iOS OS installed on the device. |
-| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk)|
+| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [Swift SDK](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/)|
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/sdk_primer.md:72]:
```suggestion
| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [Swift SDK v5.7.0+](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/)|
```

**Review by davidbielik — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/ios/storage.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ By default, Braze will collect the following [device-level properties](https://g
 * Device Locale
 * Device Model
 * Device OS Version
-* IDFV
+* IDFV (Optional with [Swift v5.7.0+](https://github.com/braze-inc/braze-swift-sdk))
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/storage.md:26]:
```suggestion
* IDFV (Optional with [iOS SDK v5.7.0+](https://github.com/braze-inc/braze-swift-sdk))
```

File: _docs/_developer_guide/platform_integration_guides/sdk_primer.md [hunk 5]
```diff
@@ -69,7 +69,7 @@ These properties are collected by the iOS SDK upon proper integration.
 | Device Locale <br> `ABKDeviceOptionLocale`| The default locale of the device. |
 | Device Model <br> `ABKDeviceOptionModel`| The specific hardware of the device.
 | Device OS Version <br> `ABKDeviceOptionOSVersion` | The version of the iOS OS installed on the device. |
-| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk)|
+| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [Swift SDK v5.7.0+](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/)|
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/sdk_primer.md:72]:
```suggestion
| Device IDFV <br> `ABKDeviceOptionIDFV`| Device identifier for vendors. IDFV collection is now optional on our [iOS SDK v5.7.0+](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/other_sdk_customizations/swift_idfv/)|
```

**Review by davidbielik — APPROVED** [review]
looks like my previous suggested changes were lost

**Review by davidbielik — APPROVED** [review]

**Review by jocelynzz — APPROVED** [review]

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**